File: src/test/java/SplittedText.java
import java.util.List;

import com.cuupa.classificator.services.stripper.TextAndPosition;

public class SplittedText {

	private List<TextAndPosition> topText;
	private List<TextAndPosition> middleText;
	private List<TextAndPosition> bottomText;

	public void setTopText(List<TextAndPosition> topText) {
		this.topText = topText;
	}

	public void setMiddleText(List<TextAndPosition> middleText) {
		this.middleText = middleText;
	}

	public void setBottomText(List<TextAndPosition> bottomText) {
		this.bottomText = bottomText;
	}

}

File: src/test/java/SplittedText.kt
import com.cuupa.classificator.knowledgebase.stripper.TextAndPosition

class SplittedText {
    private var topText: List<TextAndPosition>? = null
    private var middleText: List<TextAndPosition>? = null
    private var bottomText: List<TextAndPosition>? = null
    fun setTopText(topText: List<TextAndPosition>?) {
        this.topText = topText
    }

    fun setMiddleText(middleText: List<TextAndPosition>?) {
        this.middleText = middleText
    }

    fun setBottomText(bottomText: List<TextAndPosition>?) {
        this.bottomText = bottomText
    }
}